### DISCUSSION TITLE
Feature Request : Profile review

### DISCUSSION THREAD

**Discussion by LuffySama-Dev** [EddieHubCommunity/LinkFree#768] [Ideas]
(Profile as in the Profile pages and not just links to our repos, yt, link, etc)
Hello everyone,

I hope you are all doing great. This discussion is in regards to add a feature in our current LinkFree. 
As you know **our #eddiehub fam is growing daily**, many members are new to open source and are just getting started with it.

Most of them are trying to **create their profile** and trying to **get the Green Squares**.  

Currently they create there profiles and then share it in discord to get review. So what if we add a feature for them to add the profiles to one of our repos where our community members can review it and suggest some modifications in comments. 

Benefits : 
1] More Green squares (Commits of profiles, new issues while adding feature, Reviews in form of comments.)

2] Growth opportunity for new developers to suggest and work on how that page should be, what should it look like, it will also boost new comers confidence, also discussions among members will increase, members will work as team. 

I know it will be little complex and little tough. But our community can make it happen. 

Please let me know what do you guys think.

**Comment by suitupstan** [comment]
We already have that 🙂 https://github.com/EddieHubCommunity/support/issues/new/choose

**Comment by Vyvy-vi** [comment]
I don't think that can be equitable, with how LinkFree works
(It's supposed to be a link-sharing platform)

However, people can get review by posting an issue in the `EddieHubCommunity/support` repo, as Krish mentioned above...
